[Outlook] (manifest) update JSON support info (#3711)
OfficeDev/office-js-docs-pr: docs/outlook/online-meeting.md

If the add-in uses the Teams manifest (preview), select the **Teams Manifest (developer preview)** tab.

> [!IMPORTANT]

> > Online meeting providers aren't yet supported for the Teams manifest (preview). We are working on providing that support soon.

Reviewer: ```suggestion
> Online meeting providers aren't yet supported for the Teams manifest (preview). We're working on providing that support soon.
```
